src/AirshipBindings.iOS/ApiDefinitions.cs

```diff
@@ -2574,16 +2652,22 @@ interface UAPush
         [Export("defaultPresentationOptions", ArgumentSemantic.Assign)]
         UNNotificationPresentationOptions DefaultPresentationOptions { get; set; }
 
+        // @property (assign, readwrite, nonatomic) NSInteger badgeNumber;
+        [Export("badgeNumber", ArgumentSemantic.Assign)]
+        nint BadgeNumber { get; set; }
+
         // @property (getter=isAutobadgeEnabled, assign, readwrite, nonatomic) BOOL autobadgeEnabled;
         [Export("autobadgeEnabled")]
         bool AutobadgeEnabled { [Bind("isAutobadgeEnabled")] get; set; }
 
         // @property (readwrite, copy, nonatomic) NSArray<NSString *> *_Nonnull tags;
         [Export("tags", ArgumentSemantic.Copy)]
+        [Obsolete("Deprecated ? to be removed in SDK version 13.0. Please use the equivalent property on UAChannel.")]
         string[] Tags { get; set; }
 
         // @property (getter=isChannelTagRegistrationEnabled, assign, readwrite, nonatomic) BOOL channelTagRegistrationEnabled;
         [Export("channelTagRegistrationEnabled")]
+        [Obsolete("Deprecated ? to be removed in SDK version 13.0. Please use the equivalent property on UAChannel.")]
```

Comment:
?

Reply:
I'm not going to call out every one of these, but you get the idea. search and replace, or whatever works best.

Reply:
I appreciate you finding as many as you did!